### ISSUE TITLE
Support for RN 0.60.*

### ISSUE THREAD

**Issue by Foskas** [react-native-config/react-native-config#395]
Does this support RN 0.60/0.61 somehow ? 
I've tried and without any solution.

**Comment by saurabh1234** [comment]
I don't think it supports. You still have to link it like you do for <0.59.x and live with it. Yes you get error message which is annoying but feel free to make it 0.61.x compatible.

**Comment by blueberry6401** [comment]
I previously used react-native-config for my RN 0.60 and 0.61, follow one of the guide in this thread
https://github.com/luggit/react-native-config/issues/365
Unfortunately, I don't remember exactly what comment. Maybe you wanna try.

**Comment by charkcl** [comment]
I managed to autolink a RN 0.61 project with v0.12. Using it with fastlane, I have an example repo [here](https://github.com/charkcl/react-native-config-fastlane)
